client/landing/gutenboarding/components/mshots-image/index.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import classnames from 'classnames';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface MShotsImageProps {
+	url: string;
+	alt: string;
+	'aria-labelledby': string;
+}
+
+export function mshotsUrl( url: string, count = 0 ): string {
+	const mshotsUrl = 'https://s0.wp.com/mshots/v1/';
+	const mShotsParams = {
+		// viewport size (how much of the source page to capture)
+		vpw: 1200,
+		vph: 3072,
+		// size of the resulting image
+		w: 700,
+		h: 1800,
+	};
+	const mshotsRequest = addQueryArgs( mshotsUrl + encodeURIComponent( url ), {
+		...mShotsParams,
+		// this doesn't seem to work:
```

Comment:
have you tried `requeue: "true",`?

Reply:
No, but if that's it you may save me some time in the future :+1: